### PR TITLE
Fix model relationships

### PR DESCRIPTION
**Pull request by phinguyenphuoc**

### CODE REVIEW

**Comment by phinguyenphuoc** [comment]
image

**Comment by phinguyenphuoc** [comment]
ready

File: app/models/location.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 class Location < ApplicationRecord
-  has_many :travellings, dependent: :destroy
+  has_many :starts, class_name: "Travelling",
+    foreign_key: "location_start_id", dependent: :destroy
```
Comment (dothidiemthao) [inline app/models/location.rb]: dùng format symbol nhé :location_start_id

File: app/models/location.rb [hunk 2]
```diff
@@ -1,3 +1,7 @@
 class Location < ApplicationRecord
-  has_many :travellings, dependent: :destroy
+  has_many :starts, class_name: "Travelling",
```
Comment (dothidiemthao) [inline app/models/location.rb]: "Travelling" => dùng Travelling.name nhé
fix tương tự

**Comment by phinguyenphuoc** [comment]
ready

File: app/models/location.rb [hunk 3]
```diff
@@ -1,5 +1,8 @@
 class Location < ApplicationRecord
-  has_many :travellings, dependent: :destroy
+  has_many :travellings_start_id, class_name: Travelling.name,
+    foreign_key: :location_start_id, dependent: :destroy
+  has_many :travellings_end_id, class_name: Travelling.name,
+    foreign_key: :location_end_id, dependent: :destroy
   validates :name, uniqueness: true
   scope :ordered_by_name, ->{order("name desc")}
```
Comment (VuThiTranVan) [inline app/models/location.rb:7]: `order("name desc")` -> thành `order name: :desc`

**Pull request merged**